Adjust the mysql-log-rotate script to contain the correct log file
name for Red Hat installations.


diff -Naur mysql-5.5.20.orig/support-files/mysql-log-rotate.sh mysql-5.5.20/support-files/mysql-log-rotate.sh
--- mysql-5.5.20.orig/support-files/mysql-log-rotate.sh	2011-12-16 14:52:06.000000000 -0500
+++ mysql-5.5.20/support-files/mysql-log-rotate.sh	2012-01-26 23:24:54.402676177 -0500
@@ -3,7 +3,7 @@
 # in the [safe_mysqld] section as follows:
 #
 # [safe_mysqld]
-# err-log=@localstatedir@/mysqld.log
+# err-log=/var/log/mysqld.log
 #
 # If the root user has a password you have to create a
 # /root/.my.cnf configuration file with the following
@@ -18,7 +18,7 @@
 # ATTENTION: This /root/.my.cnf should be readable ONLY
 # for root !
 
-@localstatedir@/mysqld.log {
+/var/log/mysqld.log {
         # create 600 mysql mysql
         notifempty
 	daily
